bin/node-template/pallets/template/src/calls.rs

```diff
@@ -0,0 +1,51 @@
+use frame_support::pallet_macros::*;
+
+#[export_section]
+mod calls {
```

Comment:
Could you please add a comment that it is not possible to add that attribute on a file level like `#![export_section]`.  
I think that applies to all attribute macros.

Reply:
yes this comment would be a good idea.

And yes, custom inner attributes are a nightly-only feature atm